src/app/pages/Collectibles/CollectiblesTab/components/CollectibleItem.tsx

```diff
@@ -37,14 +32,14 @@ import { TempleChainKind } from 'temple/types';
 import { CollectibleTabSelectors } from '../selectors';
 import { toCollectibleLink } from '../utils';
 
-import { CollectibleItemImage, EvmCollectibleItemImage } from './CollectibleItemImage';
+import { TezosCollectibleItemImage, EvmCollectibleItemImage } from './CollectibleItemImage';
 
 // Fixed sizes to improve large grid performance
 const ImgContainerStyle = { width: 112, height: 112 };
-const ImgWithDetailsContainerStyle = { width: 112, height: 152 };
-const ImgStyle = { width: 110, height: 110 };
+const ImgWithDetailsContainerStyle = { width: 112, height: 132 };
```

Comment:
If you can check whether the grid performance is the same when the sizes are specified in `rem`, please do it. If it appears that it is true, please use `rem` to improve the appearance when a page is zoomed.

Reply:
did't notice any performance diffs